neural_compressor/adaptor/torch_utils/smooth_quant.py

```diff
@@ -934,18 +934,19 @@ def get_prev_absorb_layer(self, nodes, dict_child_kind, dict_child):
                 break
         return prev_absorb_layer
 
-    def get_prev_absorb_layer_helper(self, node_scopeName, dict_child_kind, dict_child, intersect):
+    def get_skipconnect_helper(self, node_scopeName, dict_child_kind, dict_child, intersect):
```

Comment:
Better change the name since we are handling many scenarios not just skip connect